warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -1184,6 +1184,18 @@ public static boolean excludeRegex(Iterable<?> values, String regex) {
         return matches;
     }
     
+    /**
+     * This is the query function to pass query option pairs to the ShardQueryConfiguration. It always evaluates to 'true' so that it does not need to be
+     * removed from the query after the args are parsed to the ShardQueryConfiguration
+     * 
+     * @param args
+     *            ignored here. used to pass key/value pairs to the query options
+     * @return true
+     */
+    public static boolean options(String[] args) {
+        return true;
```

Comment:
This seems dangerous to me in that if this is embedded with an OR, then we would return everything that gets evaluated.  Perhaps we need to actually remove this function from the jexl tree during the planning stage.

Reply:
changed visitor to return 'null' for this function, which lets the rebuilding visitor remove it from the tree